src/arch/Lights/LightsDriver.cpp

```diff
@@ -28,6 +32,11 @@ void LightsDriver::Create( const RString &sDrivers, vector<LightsDriver *> &Add
 		LOG->Info( "Lights driver: %s", Driver.c_str() );
 		Add.push_back( pDriver );
 	}
+
+	//ensure these are always available to use for debugging
+	//or if InputHandlers that want lighting state.
+	Add.push_back(new LightsDriver_SystemMessage);
+	Add.push_back(new LightsDriver_Export);
```

Comment:
LightsDriver_export.cpp is not compiled on macOS. Compiling the file unconditionally should make this work on macOS too. @DinsFire64 (pinging you because the PR is already merged)

Reply:
This is a valid point indeed